agent/go-service/aerosalvage/register.go

```
@@ -5,6 +5,7 @@ import maa "github.com/MaaXYZ/maa-framework-go/v4"
 // Register registers the Aerial Salvage custom components.
 func Register() {
 	maa.AgentServerRegisterCustomAction(aeroSalvageConfigureSwipeActionName, &ConfigureSwipeAction{})
+	maa.AgentServerRegisterCustomRecognition(aeroSalvageBalloonStateRecognitionName, &BalloonStateRecognition{})
```

golangci-lint (2.13.2)

[error] Error return value of `maa.AgentServerRegisterCustomRecognition` is not checked

(errcheck)